test/hotspot/jtreg/serviceability/jvmti/vthread/VThreadEventTest/VThreadEventTest.java

```diff
@@ -207,4 +207,3 @@ public static void main(String[] args) throws Exception {
     }
 
 }
-
```

Comment:
Does this test also need an update?
It has a line 32:
    *   -Djdk.virtualThreadScheduler.parallelism=9